crates/store/re_video/src/demux/mod.rs

```diff
@@ -138,18 +139,20 @@ impl VideoData {
     }
 }
 
-/// A segment of a video.
+/// A Group of Pictures (GOP) always starts with an I-frame, followed by delta-frames.
+///
+/// See <https://en.wikipedia.org/wiki/Group_of_pictures> for more.
```

Comment:
Nice to see that there was already a term for it that I just missed, but the name is quite unfortunate :grin: